src/types.ts

```diff
@@ -34,12 +34,12 @@ export type WalletAddress = string;
 export type ContractAddress = string;
 
 /**
- * The parameters to pass to the nft mint function.
+ * The parameters to pass to the nft mint and transfer functions.
  *
  * @beta
  */
-export type TokenMintParams = {
-  to: WalletAddress;
```

Comment:
does this not mean this becomes a breaking change? people would have to update their objects from `{to}` to `{toAddress}`?

I understand the change in itself but I don't think it's worth breaking inputs